envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -17,6 +17,11 @@ class NodeMetadata(metadata: Struct, properties: SnapshotProperties) {
         .fieldsMap["service_name"]
         ?.stringValue
 
+    val ads = metadata
+        .fieldsMap["ads"]
+        ?.boolValue
+        ?: false
```

Comment:
I think we can remove the same code from `MetadataNodeGroup` now and use this `ads` field instead